### DISCUSSION TITLE
Help. has failed with exit code 1

### DISCUSSION THREAD

**Discussion by Sliderokk** [def-studio/telegraph#481] [Q&A]
![1](https://github.com/defstudio/telegraph/assets/66663257/acf3e300-afb7-4578-9ca5-e422ecd7486c)
How to solve this problem?

**Comment by fabio-ivona** [comment]
Can't help without a relevant stack trace, sorry

**Reply by Sliderokk** [reply]
what should I do next, what alternatives, what do you recommend?

**Reply by Taurinz359** [reply]
You need to init telegram bot, here documentation how to do it
https://defstudio.github.io/telegraph/quickstart/register-new-bot

In your screen telegraph wait a token, but aborted
